src/platform/packages/shared/kbn-unified-histogram/components/chart/hooks/use_lens_props.ts

```diff
@@ -57,10 +61,11 @@ export const useLensProps = ({
         searchSessionId: request?.searchSessionId,
         getTimeRange,
         attributes,
+        esqlVariables,
```

Comment:
Looks like whenever I delete a control or change the query, 3 new requests are sent. Is it expected for this PR?